pkg/cmd/server/options.go

```diff
@@ -69,23 +93,26 @@ func (o CustomMetricsAdapterServerOptions) Config() (*apiserver.Config, error) {
 	if err := o.SecureServing.ApplyTo(&serverConfig.SecureServing, &serverConfig.LoopbackClientConfig); err != nil {
 		return nil, err
 	}
-
 	if err := o.Authentication.ApplyTo(&serverConfig.Authentication, serverConfig.SecureServing, nil); err != nil {
 		return nil, err
 	}
 	if err := o.Authorization.ApplyTo(&serverConfig.Authorization); err != nil {
 		return nil, err
 	}
-
 	if err := o.Audit.ApplyTo(serverConfig); err != nil {
 		return nil, err
 	}
+	if err := o.Features.ApplyTo(serverConfig); err != nil {
+		return nil, err
+	}
```

Comment:
Same could we have that in a separate PR since it enables new pprof flags?

Reply:
:+1: